.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Django: Run tests",
+            "type": "python",
+            "request": "launch",
+            "module": "pytest",
+            "args": [
+                "--lf", // Rerun only failed tests
```

Comment:
Doesn't this comment invalidate the json?

Reply:
Well, it's working, but you are right

Reply:
I've removed it

Reply:
Noooooooo

Reply:
Now it is